.github/workflows/pull_request.yml

```diff
@@ -1,6 +1,14 @@
 name: Build Chain
 
-on: [pull_request]
+on:
+  pull_request:
+    types: [opened, synchronize, reopened, ready_for_review]
+    paths-ignore:
+      - 'LICENSE*'
+      - '.gitignore'
+      - '*.md'
```

Comment:
There are some `.md` files on subfolders. Would this cover that?
https://github.com/kiegroup/kie-wb-distributions/blob/main/business-central-tests/README.md

Reply:
```suggestion
      - '**.md'
```